sdk/search/search-documents/README.md

```diff
@@ -25,30 +25,27 @@ Use the client library to:
 
 ## Getting started
 
-### Currently supported environments
-
-- [Node.js](https://nodejs.org/) version 8.x.x or higher
-
 ### Prerequisites
```

Comment:
Ted has a 4th H3 in his .NET readme that follows ### Authenticate the client.  It's called "### Send your first search query" and it sends a query to a prepared/loaded index on sandbox site.  This might be more than you want to do right now, but I think JS devs might find it really helpful.  Here is a link if you want to consider doing a version for JS:  https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/search/Azure.Search.Documents#send-your-first-search-query

Reply:
I like the 'send your first query' example too

Reply:
Done. Will be updated in the next commit